sale_loyalty_limit/models/sale_order.py

```diff
@@ -0,0 +1,132 @@
+from odoo import _, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _program_check_compute_points(self, programs):
+        res = super()._program_check_compute_points(programs)
+        programs_with_additional_errors = {}
```

Comment:
You already have a dict coming from `super()` just write over that one when a program can't be applied